noir-projects/noir-contracts/contracts/test_contract/src/main.nr

```diff
@@ -33,15 +36,19 @@ contract Test {
         note_getter_options::NoteStatus
     },
         deploy::deploy_contract as aztec_deploy_contract,
-        oracle::{encryption::aes128_encrypt, unsafe_rand::unsafe_rand}, utils::comparison::Comparator
+        oracle::{encryption::aes128_encrypt, unsafe_rand::unsafe_rand}, utils::comparison::Comparator,
+        macros::{storage::storage, events::event, functions::{private, public, internal}}
     };
     use dep::token_portal_content_hash_lib::{get_mint_private_content_hash, get_mint_public_content_hash};
     use dep::value_note::value_note::ValueNote;
-    use std::embedded_curve_ops::fixed_base_scalar_mul as derive_public_key;
+    // TODO investigate why the macros require EmbeddedCurvePoint and EmbeddedCurveScalar
+    use std::embedded_curve_ops::{EmbeddedCurveScalar, EmbeddedCurvePoint, fixed_base_scalar_mul as derive_public_key};
```

Comment:
Why do we have EmbeddedCurveScalar and EmbeddedCurvePoint imports here when they seem to be unused in the code?

Reply:
Noir issue, I'll create a repro asap